src/validations/rules/Writing_FedRAMP_Validations.md

```diff
@@ -0,0 +1,60 @@
+# How to create FedRAMP OSCAL validation constraints
+
+FedRAMP OSCAL documents are structured XML documents. The document syntax is defined by OSCAL XML Schema. However, XML Schema cannot express many semantic constraints, and an additional schema language is employed: Schematron.
+
+# Technologies used
+
+- [XML](https://www.w3.org/standards/xml/) is a structered document langauge.
```

Comment:
```suggestion
- [XML](https://www.w3.org/standards/xml/) is a structured document language.
```

Reply:
I'll fix this and others. I actually ran a spell checker against this. Apparently not the best of spell checkers.

Reply:
Fixed in https://github.com/18F/fedramp-automation/commit/bd71afaefcda11214aac1b0a6fd84d7f3e9a7762